server/errorHandler.ts

```diff
@@ -1,30 +1,54 @@
 import type { Request, Response, NextFunction } from 'express'
-import type { HTTPError } from 'superagent'
+import createError, { HttpError } from 'http-errors'
+import ControllerUtils from './utils/controllerUtils'
 import logger from '../log'
 
 export default function createErrorHandler(production: boolean) {
-  return (error: HTTPError, req: Request, res: Response, next: NextFunction): void => {
+  return (err: Error, req: Request, res: Response, next: NextFunction): void => {
+    if (res.headersSent) {
+      return next(err)
+    }
+
+    if (createError.isHttpError(err)) {
+      // authorization errors cause a special error page to be displayed
+      if (err.status === 403) {
+        res.status(403)
+
+        const args: Record<string, unknown> = { message: err.message }
+
+        // 403 responses from the interventions service contain further information in the
+        // response; if it's present, the authError template surfaces this to the end user
+        if (err.response) {
+          args.message = err.response.body?.message || args.message
+          args.accessErrors = err.response.body?.accessErrors
```

Comment:
As you showed in the demo, do we need to capitalize the first letter of these etc to make it a bit neater to users? Are you running these past the content team?

Reply:
yeh i'm going to do that once these changes are merged. then the next set of changes will just be cosmetic